.github/workflows/pypi-publish.yml

```diff
@@ -27,25 +27,24 @@ jobs:
 
       - name: Update version in Python package
         run: |
-          VERSION="${{ github.event.release.tag_name }}"
-          VERSION="${VERSION#v}"  # Remove 'v' prefix if present
-
-          # Check if it's a prerelease and add beta suffix
-          if [[ "${{ github.event.release.prerelease }}" == "true" ]]; then
-            # Add beta suffix if not already present
-            if [[ ! "$VERSION" =~ (a|b|rc|RC) ]]; then
-              VERSION="${VERSION}b1"
-              echo "Converting to beta version: $VERSION"
-            fi
-          fi
```

Comment:
We are doing this in the pip wrapper, so rather just pass the exact tag in to `setup.py`